### ISSUE TITLE
Self-serve forgotten password

### ISSUE THREAD

**Issue by gpimblott** [gpimblott/TechRadar#74]
Add a mechanism for a user to reset their password (email?)

**Comment by xjedam** [comment]
Yeah, email would be our best bet. Think we should start by introducing an email field to the user entity and creation/edition/etc pages as a separate pull req (so we don't run the risk of blocking either issue #74 and issue #71). This would be required for this issue, and eventually might be required for the enhanced sign-in process. Does that sound fine?

**Comment by gpimblott** [comment]
Move to AD makes this a lower priority
